webapp/src/i18n.tsx

```diff
@@ -1,44 +1,62 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import messages_ca from '../i18n/ca.json'
+import messages_de from '../i18n/de.json'
+import messages_el from '../i18n/el.json'
 import messages_en from '../i18n/en.json'
 import messages_es from '../i18n/es.json'
-import messages_de from '../i18n/de.json'
-import messages_ja from '../i18n/ja.json'
 import messages_fr from '../i18n/fr.json'
+import messages_id from '../i18n/id.json'
+import messages_it from '../i18n/it.json'
+import messages_ja from '../i18n/ja.json'
 import messages_nl from '../i18n/nl.json'
-import messages_ru from '../i18n/ru.json'
 import messages_oc from '../i18n/oc.json'
+import messages_ptBr from '../i18n/pt_BR.json'
+import messages_ru from '../i18n/ru.json'
+import messages_sv from '../i18n/sv.json'
 import messages_tr from '../i18n/tr.json'
-import messages_zhHant from '../i18n/zh_Hant.json'
 import messages_zhHans from '../i18n/zh_Hans.json'
+import messages_zhHant from '../i18n/zh_Hant.json'
 
-const supportedLanguages = ['de', 'fr', 'ja', 'nl', 'ru', 'es', 'oc', 'tr', 'zh-cn', 'zh-tw']
+const supportedLanguages = ['ca', 'de', 'el', 'en', 'es', 'fr', 'id', 'it', 'ja', 'nl', 'oc', 'pt-br', 'ru', 'sv', 'tr', 'zh-cn', 'zh-tw'] 
```

Comment:
You are adding new languages here, but you are not adding them to the Languages menus (in GlobalHeaderMenu and in SidebarMenu)